tdl-app/src/components/ApplicationBar/ApplicationBar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { AppBar, Toolbar, Typography } from '@material-ui/core';
+
+export const ApplicationBar = () => {
```

Comment:
Do we need these? 
Can these be removed if we add a index.js file to export for every component/page?

Reply:
Will be addressed later